snap/snapcraft.yaml

```diff
@@ -168,6 +237,7 @@ apps:
       # KF5_LIBEXEC_DIR: "$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/libexec/kf5"
       PATH: "$PATH:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/libexec/kf5" # Allows kioslave5 to be found, above commented env variable doesn't work for some reason  
```

Comment:
This should not be needed; libexec from the content snap should be used. Mixing qt from content snap and qt from the archives can result in issues.